### PR TITLE
Migrate Chroma to CustomComponent

### PR DESCRIPTION
**Pull request by DiogenesBR**
Migrating Chroma to CustomComponent

### CODE REVIEW

File: src/backend/langflow/components/vectorstores/Chroma.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            return Chroma.from_documents(
+                documents=documents,  # type: ignore
+                persist_directory=persist_directory if persist else None,
+                collection_name=collection_name,
+                embedding=embedding,
+                client_settings=chroma_settings,
+            )
+
+        return Chroma(
+            persist_directory=persist_directory,
```
Comment (ogabrielluiz) [inline src/backend/langflow/components/vectorstores/Chroma.py:108]: hey! Great job.

The settings should be passed here if it exists. We want to make sure Chroma can be used locally or remotely. I know it is hard to test it remotely but as long as we conform with the API we should be fine.

So, maybe check if it is not None and pass it if it exists.

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
It there's no document it should still be able to run it locally or remotely.

File: src/backend/langflow/components/vectorstores/Chroma.py (hunk 1 above)
Comment (DiogenesBR) [reply src/backend/langflow/components/vectorstores/Chroma.py:108]: The client_settings were added to the returned Chroma object, there is already a check for existing settings so it is just add the parameter here.

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Pull request merged**